doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
+There can be multiple denoising techniques.
+
+First you need to download the desired data.
+
+------------------------
+Downloading the Data Set
+------------------------
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+To fetch some specific dataset, you can use the following command::
+
+    dipy_fetch specific_dataset
+
+For example, to download sherbrooke_3shell dataset, you can use the following
+command::
+
+    dipy_fetch sherbrooke_3shell
+
+-------------------------
+Denoising using Local PCA
+-------------------------
+
+PCA-based denoising algorithms are effective denoising methods because they explore 
+the redundancy of the multi-dimensional information of diffusion-weighted datasets
+
+We are using isbi2013_2shell dataset. You can also use your own data.
+
+Following workflow requires input_files Path to the input volumes, bvalues_files Path
+to the bvalues files and bvectors_files Path to the bvectors files.
+
+Run the following workflow::
+
+    dipy_denoise_lpca /.dipy/isbi2013/phantom64.nii.gz /.dipy/isbi2013/phantom64.bval /.dipy/isbi2013/phantom64.bvec
```

Comment:
I would at least strip the heading forward slashes and maybe the dot, and maybe only put something like `data/isbi2013/phantom64.nii.gz`

Same for the rest of arguments and the rest of CLI's.

@skoudoro ?

Reply:
Addressed in d49dc04